libethereum/BlockQueue.h

```diff
@@ -67,6 +67,117 @@ enum class QueueStatus
 
 std::ostream& operator<< (std::ostream& os, QueueStatus const& obj);
 
+template<class T>
+class SizedBlockQueue
+{
+public:
+	std::size_t count() const
```

Comment:
Yeah, that's more common to our codebase, although I find one-liners less readable (and and harder debuggable)
Ok, I'll change it, fitting to a page is a good argument